index.html

```diff
@@ -1,67 +1,290 @@
+          </div>
+          <div class="intro-card">
+            <a href="#">LET’S CONNECT</a>
+            <div class="intro-card2">
+              <ul class="icon">
+                <li>
+                  <a href="#"> <img src="images/twitter.svg" /> </a>
+                </li>
+                <li>
+                  <a href="#">
+                    <img
+                      src="images/8cd679f2-0853-4869-b7c4-1c93ecf3930d.png"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/medium.svg" /> </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/github.svg" /> </a>
+                </li>
+                <li>
+                  <a href="#"> <img src="images/angellist.svg" /> </a>
+                </li>
+              </ul>
             </div>
-            
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!-- Tonic -->
+
+    <section class="grid">
+      <div class="work-card flex column">
+        <img src="images/works-img/tonic1.png" alt="Tonic" />
+        <div class="work-card1">
+          <h2 class="ton">Tonic</h2>
+          <div class="canopy">
+            <p class="canopy1">CANOPY</p>
+            <div class="round"></div>
+            <p>Back End Dev</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+
+          <div class="daily1">
+            <p class="daily">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+
+          <div class="lang1">
+            <ul class="lang">
+              <li class="lang-item">html</li>
+              <li class="lang-item">css</li>
+              <li class="lang-item1">javascript</li>
+            </ul>
+          </div>
+          <button class="project-button">See Project</button>
+        </div>
+      </div>
+
+      <!-- MULTI-POST STORIES -->
+
+      <div class="work-card flex column">
+        <img src="images/works-img/stories1.png" alt="Tonic" />
+        <div class="work-card1">
+          <h2 class="ton">Multi-Post Stories</h2>
+          <div class="canopy">
+            <p class="canopy1">CANOPY</p>
+            <div class="round"></div>
+            <p>Back End Dev</p>
+            <div class="round"></div>
+            <p>2015</p>
+          </div>
+
+          <div class="daily1">
+            <p class="daily">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+
+          <div class="lang1">
+            <ul class="lang">
+              <li class="lang-item">html</li>
+              <li class="lang-item">css</li>
+              <li class="lang-item1">javascript</li>
+            </ul>
+          </div>
+
+          <button class="project-button">See Project</button>
         </div>
```

Comment:
- [ Optional ] Please make sure that there is some space around the cards so they look more like the design
![image](https://user-images.githubusercontent.com/74744787/219809646-120f3118-47e1-4e4f-b838-d3bd748896d3.png)
![image](https://user-images.githubusercontent.com/74744787/219809893-13cd7584-ed3c-4a5c-877e-e693650fd207.png)